src/core/reducers/categories.js

```diff
@@ -31,6 +31,14 @@ type CategoryMapType = {
   },
 };
 
+export type CategorySlugType = {
+  [addonSlug: string]: ExternalCategory,
+};
+
+type AddonCategories = {
```

Comment:
I don't think we need this. It's nice that it's specific about what `AddonCategories` looks like, but the type of `categories` for an addon is already defined in `AddonType`. So instead of this you can just reference that via `$PropertyType<AddonType, 'categories'>`.

Reply:
Ohk! I don't know about this. Good to know this type of variable type checking. I will update that.